ldap.yml

```diff
@@ -0,0 +1,23 @@
+- name: Copy ldap params to conf file
+  blockinfile:
+    path: "{{ sonar_conf_file }}"
+    block: |
+         sonar.security.realm={{ sonar_security_realm }}
+         sonar.security.savePassword={{ sonar_security_savePassword }}
+         ldap.url={{ ldap_url }}                                           
+         ldap.authentication={{ ldap_authentication }}
+         ldap.bindDn={{ ldap_bindDn }}                    
+         ldap.bindPassword={{ ldap_bindPassword }}
+         ldap.user.baseDn={{ ldap_user_baseDn }}
+         ldap.user.request={{ ldap_user_request }}
+    backup: yes
+
+- name: Restart Sonar service
+  service: name=sonar state=restarted enabled=yes              
+                                                             
+- name: Ensure LDAP is configured well
```

Comment:
I think this should become ansible fact instead, thus you will also gain idempotency